src/next.config.js

```diff
@@ -67,7 +67,7 @@ const nextConfig = {
         },
       },
       {
-        test: /static\/.*\.(css)$/,
+        test: /static\/.*\.(css|json)$/,
```

Comment:
Doesn't hurt but don't think it's needed.

Reply:
@znarf I was hoping to solve the issue with now deployment for the manifest.json. It is not served: https://opencollective-frontend-oqeepovgza.now.sh/static/manifest.json

Do you have any ideas how to fix that?

Reply:
The `express` route should be enough, not sure why it's "Not Found" at the moment. Does it work locally?

Reply:
@znarf It works locally when I run `npm run dev`

Reply:
Got the error in logs:

```
Error: ENOENT: no such file or directory, stat '/usr/src/frontend/dist/static/manifest.json'
error: middlewareError {"meta":{"error":{"errno":-2,"code":"ENOENT","syscall":"stat","path":"/usr/src/frontend/dist/static/manifest.json","expose":false,"statusCode":404,"status":404},"level":"error","message":"uncaughtException: ENOENT: no such file or directory, stat '/usr/src/frontend/dist/static/manifest.json'\nError: ENOENT: no such file or directory, stat '/usr/src/frontend/dist/static/manifest.json'","stack":"Error: ENOENT: no such file or directory, stat '/usr/src/frontend/dist/static/manifest.json'","exception":true,"date":"Thu Jun 20 2019 12:03:44 GMT+0000 (Coordinated Universal Time)","process":{"pid":19,"uid":0,"gid":0,"cwd":"/usr/src/frontend","execPath":"/usr/local/bin/node","version":"v11.8.0","argv":["/usr/local/bin/node","/usr/src/frontend/dist/server"],"memoryUsage":{"rss":103780352,"heapTotal":62373888,"heapUsed":35859328,"external":3735538}},"os":{"loadavg":[2.30810546875,2.00439453125,1.70263671875],"uptime":10474299},"trace":[],"req":{"url":"/static/manifest.json","headers":{"host":"opencollective-frontend-dijdcbeahl.now.sh","x-forwarded-host":"opencollective-frontend-dijdcbeahl.now.sh","x-real-ip":"91.64.96.86","x-forwarded-for":"91.64.96.86","x-forwarded-proto":"https","x-now-id":"cbjnw-1561032224544-5875e57f260d","x-zeit-co-forwarded-for":"91.64.96.86","connection":"close","user-agent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:67.0) Gecko/20100101 Firefox/67.0","accept":"text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8","accept-language":"fr-FR,fr;q=0.8,en;q=0.5,en-US;q=0.3","accept-encoding":"gzip, deflate, br","dnt":"1","upgrade-insecure-requests":"1","x-now-trace":"bru1,sfo1"},"method":"GET","httpVersion":"1.1","originalUrl":"/static/manifest.json","query":{}}}}

--
 

Reply:
I guess, to reproduce, `npm run build` then `npm run start`

Reply:
See: https://github.com/opencollective/opencollective-frontend/blob/master/scripts/build_server.sh

In the end, I would be ok to skip the `manifest.json` and just inject the JS object in the route. Less friction, and allow us to add dynamic data in the future.

```
const manifest = { ... };
res.send(manifest)
```

Reply:
@znarf I updates routes to serve the manifest object directly. This works now on now deployment. 

For the whole flow, the api PR needs to be deployed :-)